### DISCUSSION TITLE
Wildcards

### DISCUSSION THREAD

**Discussion by SysMan-One** [variar/klogg#540] [Ideas]
Hello!
Please return back that searching variant is exist in the glogg - wildcards. 
It's not very useful to break fingers on  typing regexp-like scripting .
3xPlease!

**Comment by variar** [comment]
The wildcards search internally just did replacement of `?` to `.` and `*` to `.*`. Using regex that is one extra character to type in case a `*` wildcard is required. And that internal replacement did mess up with escaping in some case. I'm not convinced that there is a big use case for that.
